src/engine/graphics.lua

```diff
@@ -46,5 +46,5 @@ end
 -- Set color with HSV values
 function love.graphics.setColorHSV(h, s, v, a)
     local r, g, b = std.hsvToRgb(h, s, v)
-    love.graphics.setColor(r, g, b, a or 255)
+    love.graphics.setColor(r / 255, g / 255, b / 255, a and a / 255 or 1)

```

Comment:
@bhaskar10h, Let me know if you want me to include a way to work with HSL here.

Reply:
@bhaskar10h, I may also include CMYK if you want me to.

Reply:
> @bhaskar10h, Let me know if you want me to include a way to work with HSL here.

CMYK is too much for this project and I don't want  this to be bright vibrant.